src/rocm_docs/core.py

```diff
@@ -144,7 +144,7 @@ def _set_article_info(app: Sphinx, _: Config) -> None:
     if app.config.setting_all_article_info is False and len(app.config.article_pages) == 0:
         return
 
-    with open("_templates/components/article-info.html", "r") as file:
+    with open("../_readthedocs/html/article-info.html", "r") as file:
```

Comment:
There's a lot of things that could be cleaned up in this article info handling, but I'm gonna focus on your issue at hand:

You should be able to get a handle to the file from the installed location of rocm-docs-core using `importlib.resources`, you can see how its used in the `doxygen` module for copying the files here: https://github.com/RadeonOpenCompute/rocm-docs-core/blob/d7088ae2760b143936f47479d397f9781a7e4c3d/src/rocm_docs/doxygen.py#L60

The returned object behaves similarly to `pathlib.Path` so if you just subpath it to the correct file `pkg / "rocm_docs_theme/components/article-info.html"` then it is like if you had a path to that file. 
**Note that you shouln't need to copy it, just `.open()` the returned object and read its content.**

Reply:
Thanks, I have made the change in 50c55f2